heareval/predictions/task_predictions.py

```diff
@@ -897,22 +900,31 @@ def print_scores(mode, scores_and_trainers):
             deterministic=deterministic,
         )
         scores_and_trainers.append(
-            (best_model_score, model_path, epoch, trainer, hparams, postprocessing)
+            (
+                best_model_score,
+                model_path,
+                epoch,
+                time_in_min,
+                trainer,
+                hparams,
+                postprocessing,
+            )
         )
         print_scores(mode, scores_and_trainers)
 
     with open(embedding_path.joinpath("valid.hparams.jsonl"), "wt") as w:
-        for score, _, _, _, hparams, _ in scores_and_trainers:
+        for score, _, _, _, _, hparams, _ in scores_and_trainers:
             w.write(json.dumps([score, hparams_to_json(hparams)]) + "\n")
     with open(embedding_path.joinpath("valid.postprocessing.jsonl"), "wt") as w:
-        for score, _, _, _, _, postprocessing in scores_and_trainers:
+        for score, _, _, _, _, _, postprocessing in scores_and_trainers:
             w.write(json.dumps([score, postprocessing]) + "\n")
 
     # Use that model to compute test scores
     (
         best_score,
         best_model_path,
-        best_model_epoch,
+        best_epoch,
+        best_time_in_min,
```

Comment:
I might be reading this wrong, but why it scores_and_trainers[0] the best? Isn't it just the first grid point? I'm not seeing where scores_and_trainers is sorted based on scores before this selection